packages/api-reference/src/features/InfoObject/Contact.vue

```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+import { type OpenAPIV3_1 } from '@scalar/openapi-types'
+
+defineProps<{
+  value?: OpenAPIV3_1.ContactObject
+}>()
+</script>
+
+<template>
+  <template v-if="value">
+    <div class="flex gap-1">
+      <a :href="`mailto:${value?.email}`">{{ value.name }}</a>
```

Comment:
we might only have an email or only have a name, it should work in both cases

and there’s also an URL

<img width="993" height="296" alt="Screenshot 2025-07-17 at 16 48 13" src="https://github.com/user-attachments/assets/d5a623c4-83fa-44a2-9497-f6b2813b1e94" />